sunpy/net/fido_factory.py

```diff
@@ -183,7 +184,11 @@ def __str__(self):
         for block in self:
             ret += f"{len(block)} Results from the {block.client.__class__.__name__}:\n"
             if block.client.info_url is not None:
-                ret += f'Source: {block.client.info_url}\n\n'
+                ret += f'Source: {block.client.info_url}\n'
+            size = block.total_size()
+            if np.isfinite(size):
+                ret += f'Total size: {size}\n'
```

Comment:
The VSO isn't always reliable at accurately reporting size, and some clients wont contribute, so I think we should hedge the language a little here?
```suggestion
                ret += f'Total estimated size: {size}\n'
```